book/src/ed_overview.md

```diff
@@ -2,15 +2,15 @@
+These protocol-based rewards, to be distributed to participating validation and replication clients, are to be a result of a global supply inflation rate, calculated per Solana epoch and distributed amongst the active validator set. As discussed further below, the per annum inflation rate is based on a pre-determined disinflationary schedule. This provides the network with monetary supply predictability which supports long term economic stability and security. 
 
-Transaction fees are market-based participant-to-participant transfers, attached to network interactions as a necessary motivation and compensation for the inclusion and execution of a proposed transaction (be it a state execution or proof-of-replication verification). A mechanism for continuous and long-term funding of the mining pool through a pre-dedicated portion of transaction fees is also discussed below.
+Transaction fees are market-based participant-to-participant transfers, attached to network interactions as a necessary motivation and compensation for the inclusion and execution of a proposed transaction (be it a state execution or proof-of-replication verification). A mechanism for continuous and long-term economic stability through partial burning of each transaction fee is also discussed below.
 
 A high-level schematic of Solana’s crypto-economic design is shown below in **Figure 1**. The specifics of validation-client economics are described in sections: [Validation-client Economics](ed_validation_client_economics.md), [State-validation Protocol-based Rewards](ed_vce_state_validation_protocol_based_rewards.md), [State-validation Transaction Fees](ed_vce_state_validation_transaction_fees.md) and [Replication-validation Transaction Fees](ed_vce_replication_validation_transaction_fees.md). Also, the chapter titled [Validation Stake Delegation](ed_vce_validation_stake_delegation.md) closes with a discussion of validator delegation opportunties and marketplace. The [Replication-client Economics](ed_replication_client_economics.md) chapter will review the Solana network design for global ledger storage/redundancy and replicator-client economics ([Storage-replication rewards](ed_rce_storage_replication_rewards.md)) along with a replicator-to-validator delegation mechanism designed to aide participant on-boarding into the Solana economy discussed in [Replication-client Reward Auto-delegation](ed_rce_replication_client_reward_auto_delegation.md). The [Economic Sustainability](ed_economic_sustainability.md) section dives deeper into Solana’s design for long-term economic sustainability and outlines the constraints and conditions for a self-sustaining economy. An outline of features for an MVP economic design is discussed in the [Economic Design MVP](ed_mvp.md) section. Finally, in chapter [Attack Vectors](ed_attack_vectors.md), various attack vectors will be described and potential vulnerabilities explored and parameterized.
 
 <!-- ![img alt text](solana_economic_design.png) -->
```

Comment:
I wanted to clarify a couple of things about how the PoRep rewards are setup right now using the diagram below as a reference: 
- In the replicator->validator fees portion; Currently, PoRep tx fees are not pooled or treated, in anyway, differently from a regular tx.
- Replicators and validators are both rewarded "storage points" and the point values are calculated based on replicator/validator participation in PoRep (similar to stake-validation rewards)

So Validators' PoRep rewards are distributed based on `# validated proofs` . Replicators' PoRep rewards are distributed based on `# valid proofs submitted`.

Does that seem reasonable instead of what's in the diagram ?

Reply:
> In the replicator->validator fees portion; Currently, PoRep tx fees are not pooled or treated, in anyway, differently from a regular tx.

The original idea to isolate validator PoRep verification rewards from the mining pool was more of an instinct on my part. I.e. There are a lot of interesting validator-replicator collusion opportunities in this relationship and the thought was that by separating the two reward paths we might mitigate some of these.  I think this will be something to keep an eye on and potentially consider sequestering PoRep transaction fees for a later date.

@sakridge  WDYT on this point?

> So Validators' PoRep rewards are distributed based on # validated proofs . Replicators' PoRep rewards are distributed based on # valid proofs submitted.

One of the challenges is that validators can't be caught (through on-chain computation) claiming real PoReps are fake (i.e. censoring).  Because of this, I think we should have a slight disincentive to playing fast-and-loose with rejecting PoReps and I think that rewarding validators in proportion to `#of validated PoReps - # of invalid PoReps`, will be desirable.

How does that sound?

Reply:
> The original idea to isolate validator PoRep verification rewards from the mining pool was more of an instinct on my part.

The rewards are separated, the tx fees are not.

> One of the challenges is that validators can't be caught (through on-chain computation) claiming real PoReps are fake (i.e. censoring).

Yeah the rewards scheme will need to evolve as we flesh out the PoRep implementation. I was only describing what we have right now. But yes, the final implementation must factor in the challenges from replicators.

Reply:
> > In the replicator->validator fees portion; Currently, PoRep tx fees are not pooled or treated, in anyway, differently from a regular tx.
> 
> The original idea to isolate validator PoRep verification rewards from the mining pool was more of an instinct on my part. I.e. There are a lot of interesting validator-replicator collusion opportunities in this relationship and the thought was that by separating the two reward paths we might mitigate some of these. I think this will be something to keep an eye on and potentially consider sequestering PoRep transaction fees for a later date.
> 
> @sakridge WDYT on this point?
> 
> > So Validators' PoRep rewards are distributed based on # validated proofs . Replicators' PoRep rewards are distributed based on # valid proofs submitted.
> 
> One of the challenges is that validators can't be caught (through on-chain computation) claiming real PoReps are fake (i.e. censoring). Because of this, I think we should have a slight disincentive to playing fast-and-loose with rejecting PoReps and I think that rewarding validators in proportion to `#of validated PoReps - # of invalid PoReps`, will be desirable.
> 
> How does that sound?

@sagar-solana just checking we're okay with this?
@sakridge thoughts?

Reply:
Hm, I'm not entirely sure if we just want to `#of validated PoReps - # of invalid PoReps`. I think it should be something like `#of validated PoReps - # of PoReps that mismatch with the replicators challenge`. Basically the replicator will eventually reveal which proofs were real and which ones were challenges, this lets the replicator mark negative points on the validator if it gets things wrong (all done by the program ofcourse).

Reply:
`# of validated PoReps - # of invalid PoReps` protects against replicators against a censoring Validator.  
When you talk about "PoReps that mismatch with replicators challenge", I'm not sure I understand how a 'final verdict' can be made on any PoRep.  It seems like you'll always end up having to trust the last validator that verifies.  Can you walk me through how your suggestion mitigates this situation? It would look something like:

1. Validator censors by claiming certain replicators PoReps false
2. Censored replicators submit challenge
3. ?  This is where I could use your help. If these challenges go back to the same validator, then I don't think we get anywhere, and penalizing the validator for a "mismatch" seems like it has opened up an attack vector for replicators.  If the challenges go to a different validator (assuming that we have validators available with overlapping samples), then we're just trusting that validator's (who could also be censoring) verdict, no? 

Not sure I'm thinking of this correctly, let me know what you think :)

Reply:
The PoRep game is described in the book. https://github.com/solana-labs/solana/blob/master/book/src/ledger-replication.md#the-porep-game

Basically, there is a period of time(a turn) after the replicators submit proofs (fake proofs, a.k.a challenges will be part of this) for validators to verify those proofs. Following the validators' verifications, there is another turn during which the replicator reveals which proofs were real and which were not (it has to prove to the program the fake ones were really fake). 

This allows the replicator's challenge stage (when it shows its hand) to automatically discredit validators that incorrectly responded to those proofs. 
Now since proof validation is randomized (i.e, validators are not allowed to choose which replicator and which set of proofs from a replicator's turn are picked) if a validator is censoring a replicator it should be an outlier compared to other validators that also looked at proofs from a replicator during that turn. 

Basically unless most validators are censoring replicators, we should be able to discount the validator that tried to censor. There were plans to stake weight this as well but I don't think the book captures that. 

Does that help ? It doesn't directly target the problem you described but it should provide some protection

Reply:
> Now since proof validation is randomized (i.e, validators are not allowed to choose which replicator and which set of proofs from a replicator's turn are picked) if a validator is censoring a replicator it should be an outlier compared to other validators that also looked at proofs from a replicator during that turn.

Ah, okay this is the bit that I was missing. I think that all makes sense then.  It would be some off-chain metric or monitoring that might flag something like this. 

I'm cool starting with `#of validated PoReps - # of PoReps that mismatch with the replicators challenge` then and seeing how far we get with it!